tests/unit/core/components/TestInstallButton.js

```diff
@@ -201,7 +216,9 @@ describe('<InstallButton />', () => {
 
     Simulate.click(installButton);
 
-    expect(fakeLog.info.firstCall.args[0]).toEqual('Adding OpenSearch Provider');
-    expect(fakeWindow.external.AddSearchProvider.firstCall.args[0]).toEqual(fakeAddon.installURL);
+    expect(fakeLog.info.firstCall.args[0]).toEqual(
```

Comment:
If you want to split expect calls onto multiple lines putting the `.toEqual` on the next line might be more natural.